File: Easy/Java/2535. Difference Between Element Sum and Digit Sum of an Array.java
class Solution {
    public int differenceOfSum(int[] nums) {
        int digitSum = 0;
        int arraySum = 0;

        for(int i = 0; i < nums.length; i++){
            arraySum += nums[i];
            while(nums[i] != 0){
                digitSum += nums[i] % 10;
                nums[i] /= 10;
            }
        }

        return Math.abs(digitSum - arraySum);
    }
}

File: Easy/JavaScript/2535. Difference Between Element Sum and Digit Sum of an Array.js
var differenceOfSum = function(nums) {
    let digitSum = 0;
    let arraySum = 0;

    for(let i = 0; i < nums.length; i++){
        arraySum += nums[i];
        while(nums[i] != 0){
            digitSum += nums[i] % 10;
            nums[i] = Math.floor(nums[i] / 10);
        }
    }

    return Math.abs(digitSum - arraySum);
};
